### PR TITLE
Fix run scripts and add TestOperatorUpgrade function for 1.1.x

### LINKED ISSUE #560: Automates cluster upgrade test
Automates cluster upgrade test not in CI, just for manual testing
The test must accept source and target version parameters, checkout source branch or tag, deploys operator version, create a cluster and than upgrade it to the target version.
#575 need to be resolved before
#576 need to be resolved before
#579 need to be resolved before

### PR DESCRIPTION
**Pull request by dmvolod**
Preparation for fix #560

### CODE REVIEW

**Comment by ryanemerson** [comment]
@rigazilla Do we still want fixes for `1.1.x`? I assumed that it's not necessary now that `2.0.x` is what's being consumed downstream.

**Comment by dmvolod** [comment]
> @rigazilla Do we still want fixes for `1.1.x`? I assumed that it's not necessary now that `2.0.x` is what's being consumed downstream.

It's just for upgrade testing purposes

**Comment by ryanemerson** [comment]
Sure, but if we're no longer supporting this branch then there is no need for this fix or the follow up. I just wanted to clarify this so that you don't waste anytime creating unnecessary PRs

**Comment by dmvolod** [comment]
> Sure, but if we're no longer supporting this branch then there is no need for this fix or the follow up. I just wanted to clarify this so that you don't waste any time creating unnecessary PRs

I hope it will last PR for the 1.1.x :smile: 
It supported and maintained yet, please see related article
https://access.redhat.com/articles/4933551
8.0 is 1.1.x branch
8.1 is 2.0.x branch
If someone customer or client happens migration problem from 1.1.x or 8.0 it would be nice to have common automation tool for upgrade task rather than waste time for manual testing for problem detection. Many of the issues come from the automated tests and can't handle manually.

File: test/e2e/main_test.go [hunk 1]
```diff
@@ -24,60 +24,74 @@ import (
...
+	SinglePodTimeout     = 5 * time.Minute
+	RouteTimeout         = 240 * time.Second
+	DefaultPollPeriod    = 1 * time.Second
+
+	OperatorUpgradeStageNone = "NONE"
+	OperatorUpgradeStageFrom = "FROM"
+	OperatorUpgradeStageTo   = "TO"
+)
+
+var (
```
Comment (ryanemerson) [inline test/e2e/main_test.go:38]: This was the case before, but is there any reason why these are vars and not constants? I haven't checked everyone, but it seems in most cases the values are never updated after creation.

**Comment by ryanemerson** [comment]
Thanks @dmvolod

**Pull request merged**